budoux/feature_extractor.py

```diff
@@ -20,13 +20,13 @@
 import sys
 import typing
 
-from .utils import SEP, Result
+from .utils import SEP, INVALID, Result
 
 with open(os.path.join(os.path.dirname(__file__), 'unicode_blocks.json')) as f:
   block_starts: typing.List[int] = json.load(f)
 
 
-def unicode_block_index(w: str) -> int:
+def unicode_block_feature(w: str) -> str:
   """Returns the index of the Unicode block that the character belongs to.
```

Comment:
Please update the comment too.

Reply:
Actually, `unicode_block_index` sounds right to me here since it's more of a "stringified" Unicode block index rather than what we call a Unicode feature throughout this library (e.g. "UB:108"). Reverting the function name instead of updating the docstring. I will formalize the terminologies more strictly in the following PRs, too.